configuration.rst

```diff
@@ -66,20 +66,13 @@ readable. These are the main advantages and disadvantages of each format:
 
 * **YAML**: simple, clean and readable, but not all IDEs support autocompletion
   and validation for it. :doc:`Learn the YAML syntax </reference/formats/yaml>`;
```

Comment:
todo: update to mention json schemas

Reply:
Just asking: these TODOs are for this PR or for future PRs? Thanks.

Reply:
In this PR, just pushed them :)

(I quickly opened this wip branch to better show what I want, and then discovered these 2 items afterwards)